CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 Change Log
 ==========
+
+Version 1.2.2 *(2019-1-11)*
+----------------------------
+* Changed: Don't automatically create lint baseline file. Projects can manually specify a baseline file in `lintOptions` if they want it.
```

Comment:
It might be nice to include more explicit instructions about how to do this in the README.

Reply:
I added an example in the changelog